src/definitions/modules/search.js

```diff
@@ -329,8 +329,10 @@
                             ;
                             $result
                                 .removeClass(className.active)
+                                .trigger('focusout')
```

Comment:
Do we really need this? The direct focusin trigger 3 lines below should automatically trigger the focusout form the previous focused element, isnt it?